tests/end-to-end/api/test.ts

```diff
@@ -1,11 +1,9 @@
 import { expect } from '@playwright/test';
 import { test, mockApiFail, mockApiSuccess } from '../custom-context';
 import { picsureUser, roles as mockRoles, mockExpiredToken, mockToken } from '../mock-data';
-import type { Branding } from '../../../src/lib/configuration';
-import * as config from '../../../src/lib/assets/configuration.json' assert { type: 'json' };
-//TypeScript is confused by the JSON import so I am fxing it here
-/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-const branding: Branding = JSON.parse(JSON.stringify((config as any).default));
+import type { Branding } from '../../../src/lib/models/Configuration';
+import brandingJson from '../../../src/lib/assets/configuration.json' assert { type: 'json' };
```

Comment:
I forgot about this, I think `assert` was deprecated because of some security issue.
```suggestion
import brandingJson from '../../../src/lib/assets/configuration.json' with { type: 'json' };
```